### PR TITLE
feat: add a helper to know automation reached goal state

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

Adds a new helper to check automation status 

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -36,8 +36,8 @@ lint: ## Run linter
 .PHONY: check
 check: test lint ## Run tests and linters
 
-.PHONY: addcopy
-addcopy:
+.PHONY: addlicense
+addlicense:
```
Comment (gssbzn) [inline Makefile:40]: drive by, the githook commit had the wrong name

File: atmcfg/fixtures.go [hunk 2]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2021 MongoDB Inc
```
Comment (gssbzn) [inline atmcfg/fixtures.go:1]: drive by now the the githook is fixed

File: opsmngr/automation_status.go [hunk 3]
```diff
@@ -57,8 +57,12 @@ type AutomationStatus struct {
 }
 
 type ProcessStatus struct {
-	Plan                    []string `json:"plan"`
+	ErrorCode               *int     `json:"errorCode,omitempty"`
+	ErrorCodeDescription    *string  `json:"errorCodeDescription,omitempty"`
+	ErrorCodeHumanReadable  *string  `json:"errorCodeHumanReadable,omitempty"`
+	ErrorString             *string  `json:"errorString,omitempty"`
```
Comment (threebee) [inline opsmngr/automation_status.go:63]: [q] Is this change related?

**Review by threebee — APPROVED** [review]
LGTM! I have added a question.
